packages/trader/src/AppV2/Components/TradeParameters/TakeProfit/take-profit-header.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { ActionSheet } from '@deriv-com/quill-ui';
+import { LabelPairedArrowLeftMdRegularIcon, LabelPairedCircleInfoMdRegularIcon } from '@deriv/quill-icons';
+import { Localize } from '@deriv/translations';
+
+type TAllowEqualsProps = {
+    current_index: number;
+    onNextClick: () => void;
+    onPrevClick: () => void;
+};
+
+const TakeProfitHeader = ({ current_index, onNextClick, onPrevClick }: TAllowEqualsProps) => (
```

Comment:
The component is almost same as allow-equal-header.tsx . Shall we create a commmon component and extend that .

Reply:
Sounds good) Let me create

Reply:
I think I can even integrate it inside of Carousel itself

Reply:
Actually In my case the working is a little different . So maybe can create it inside carousel but should allow it to overwrite if external header provided .

Reply:
Okay, then better outside of Carousel)